tests/test_infer.py

```diff
@@ -1,8 +1,11 @@
-import pytest
-import pyhf
+from copy import deepcopy
+
 import numpy as np
+import pytest
 import scipy.stats
 
+import pyhf
+
 
 @pytest.fixture(scope='module')
```

Comment:
```suggestion
@pytest.fixture(scope='function')
```

Reply:
That's fine. I hadn't done this to avoid having to have the fixture evaluate as often, but this is actually using the scoping for how it was intended, so that's good.